pycytominer/tests/test_cyto_utils/test_util.py

```diff
@@ -144,3 +146,42 @@ def test_get_pairwise_correlation():
     expected_result = -0.8
     x_y_cor = pair_df.query("correlation != 0").round(1).correlation.values[0]
     assert x_y_cor == expected_result
+
+
+def test_check_fields_of_view():
+    data_fields_of_view = [1, 3, 4, 5]
+
+    valid_input_fields_of_view = [1, 3, 4]
+    assert check_fields_of_view(data_fields_of_view, valid_input_fields_of_view) is None
```

Comment:
```suggestion
    assert check_fields_of_view(data_fields_of_view, valid_input_fields_of_view)
```

I think you can drop `is None`, but I'm not 100% sure

Reply:
Tried dropping `is None` the test failed. So leaving it as is.